lib/adapters/babel/6.x.coffee

```diff
@@ -43,6 +43,10 @@ class Babel extends Adapter
 
     data = { result: res.code }
     if res.map
+      # Convert source to absolute path.
+      # This is done for consistency with other accord adapters.
+      if res.map.sources then res.map.sources[0] = res.options.filename
```

Comment:
I'm not aware of any circumstances where there will be more than one source for a Babel file but I don't know the answer to this for sure.

Another possible appoach:
```coffee
dirname = path.dirname(res.options.filename)
res.map.sources = res.map.sources.map (source) -> path.join(dirname, source)
```